### PR TITLE
used media query to make my website responsive

### PR DESCRIPTION
**Pull request by Tracy-miranja**
used media query @ min-width 768px, to make my portfolio responsive on desktops and tablets.
I added the whole page: The header, body, and footer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,15 +72,14 @@ <h1>I’m Tracy<br />Glad to see,You!</h1>
...
               <li>2015</li>
             </ul>
           </div>
           <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
           </p>
           <div class="tags">
             <ul>
```
Comment (ghost) [inline index.html:85]: - [ ] Kindly include the languages used in your project cards and also implement the `hover`  functionality to your buttons to match the original design

image

- [ ] Kindly note that in your `about me` section, elements in the language cards are supposed to be aligned vertically and not horizontally

image

### Checkout the original design

image

- [ ] Kindly adjust the height of your inputs in the contact section to match the original design

image

### Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Tracy-miranja , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
- [ ] Kindy be sure to include the `key features` header in your readme file to highlight the key features of your project. Also, add `MIT` license to your readme to make it more professional.

_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (RaoAkif) [reply index.html:85]: [OPTIONAL] You are using `.sub-lang a` class for its styling. Kindly edit this span element according to the FIGMA design. According to Figma:
```
font-weight: 500;
font-size: 15px;
```
image

Your design looks like this:
image

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @Tracy-miranja, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions
- Update the styling of `.sub-lang a` or create a different `id` as decribe in the following comment.
- _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**